Задание 6.1/Задание 6.1/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Задание_6._1
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите текст на русском языке:");
+            string russianText = Console.ReadLine();
+            Console.WriteLine("Ваш текст в транслитерации: " + RusToTrans(russianText));
+        }
+        static string RusToTrans(string Texttt)
+        {
+            Texttt = Texttt.ToUpper();
+            return Texttt.Replace("А", "A")
```

Comment:
Локальные и параметры методов в C# переменные принято называть с маленькой буквы. И можно было проще
```suggestion
        static string RusToTrans(string texttt)
        {
            return texttt.ToUpper()
            .Replace("А", "A")
```